File: Folder CPP/Produk.cpp
#include <iostream>
#include <string>
using namespace std;

class Produk
{
protected:
    string kode, nama;
    int harga;

public:
    Produk(string kode, string nama, int harga)
        : kode(kode), nama(nama), harga(harga) {}

    string getKode() const { return kode; }
    string getNama() const { return nama; }
    int getHarga() const { return harga; }
};

File: Folder Java/Produk.java
public class Produk {
    protected String kode, nama;
    protected int harga;

    public Produk(String kode, String nama, int harga) {
        this.kode = kode;
        this.nama = nama;
        this.harga = harga;
    }

    public String getKode() { return kode; }
    public String getNama() { return nama; }
    public int getHarga() { return harga; }
}